homestyle.css

```diff
@@ -10,7 +12,7 @@ body {
 	margin: 0; 
 	padding: 0;
 	font-family: 'Montserrat', sans-serif;
-	background-color: black;
+	background-color: #000000;
 	color: #ffffff;
```

Comment:
I think hexadecimal colors are generally written in uppercase by convention. 
Also you can shorten white and black to 
```
#000
#FFF
```